actions/setup-keeper/action.yml

```diff
@@ -7,14 +7,18 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: "3.10.6"
     - name: install keepercommander
       shell: bash
-      run: pip3 install cryptography==37.0.2 pyOpenSSL==22.0.0 keepercommander==16.6.5
+      run: |
+        python3 -m pip install --upgrade pip==22.2.2
+        python3 -m pip install cryptography==37.0.2 pyOpenSSL==22.0.0 keepercommander==16.6.5
```

Comment:
Make sure that the pip we are referencing is tied to the python executable version we are running in the action instead of using the pip3 alias.